doc/dependencies.md

```diff
@@ -1,51 +1,53 @@
-| libpng |  |  |  |  | [Yes](https://github.com/bitcoin/bitcoin/blob/master/depends/packages/qt.mk) |
-| librsvg | |  |  |  |  |
-| MiniUPnPc | [2.2.2](https://miniupnp.tuxfamily.org/files) |  | No |  |  |
-| PCRE |  |  |  |  | [Yes](https://github.com/bitcoin/bitcoin/blob/master/depends/packages/qt.mk) |
-| Python (tests) |  | [3.6](https://www.python.org/downloads) |  |  |  |
-| qrencode | [3.4.4](https://fukuchi.org/works/qrencode) |  | No |  |  |
-| Qt | [5.12.11](https://download.qt.io/official_releases/qt/) | [5.9.5](https://github.com/bitcoin/bitcoin/issues/20104) | No |  |  |
-| SQLite | [3.32.1](https://sqlite.org/download.html) | [3.7.17](https://github.com/bitcoin/bitcoin/pull/19077) |  |  |  |
-| XCB |  |  |  |  | [Yes](https://github.com/bitcoin/bitcoin/blob/master/depends/packages/qt.mk) (Linux only) |
-| systemtap ([tracing](tracing.md))|  |  |  |  | |
-| xkbcommon |  |  |  |  | [Yes](https://github.com/bitcoin/bitcoin/blob/master/depends/packages/qt.mk) (Linux only) |
-| ZeroMQ | [4.3.1](https://github.com/zeromq/libzmq/releases) | 4.0.0 | No |  |  |
-| zlib |  |  |  |  | [Yes](https://github.com/bitcoin/bitcoin/blob/master/depends/packages/qt.mk) |
-
-<a name="note1">Note \*</a> : When compiling with `-stdlib=libc++`, the minimum supported libc++ version is 7.0.
-
-Controlling dependencies
-------------------------
-Some dependencies are not needed in all configurations. The following are some factors that affect the dependency list.
-
-#### Options passed to `./configure`
-* MiniUPnPc is not needed with `--without-miniupnpc`.
-* libnatpmp is not needed with `--without-natpmp`.
-* Berkeley DB is not needed with `--disable-wallet` or `--without-bdb`.
-* SQLite is not needed with `--disable-wallet` or `--without-sqlite`.
-* Qt is not needed with `--without-gui`.
-* If the qrencode dependency is absent, QR support won't be added. To force an error when that happens, pass `--with-qrencode`.
-* If the systemtap dependency is absent, USDT support won't compiled in.
-* ZeroMQ is needed only with the `--with-zmq` option.
-
-#### Other
-* librsvg is only needed if you need to run `make deploy` on (cross-compilation to) macOS.
-* Not-Qt-bundled zlib is required to build the [DMG tool](../contrib/macdeploy/README.md#deterministic-macos-dmg-notes) from the libdmg-hfsplus project.
+# Dependencies
+
+These are the dependencies currently used by Bitcoin Core.
+You can find installation instructions in the `build-*.md` file for your platform.
+"Runtime" and "Version Used" are both in reference to the release binaries.
+
+## Build, Debug & Tests
+
+A compiler with support for C++17 and `std::filesystem` is required.
+
+When compiling against `-stdlib=libc++`, version 7.0 or later is required.
+
+| Compiler | Minimum required |
+| --- | --- |
+| [Clang](https://clang.llvm.org) | [7.0](https://github.com/bitcoin/bitcoin/pull/23060) |
```

Comment:
I'd love tables free from links:
```
| Compiler   | Minimum required |
| ---------- | ---------------- |
| [Clang][0] | [7.0][1]         |
```

```
[0]: https://clang.llvm.org
[1]: https://github.com/bitcoin/bitcoin/pull/23060
```

Much better than:
```
| Compiler | Minimum required |
| --- | --- |
| [Clang](https://clang.llvm.org) | [7.0](https://github.com/bitcoin/bitcoin/pull/23060) |
```

Reply:
Not necessary IMO, and matching up the indexes may get confusing if there is a long list of footnotes. Might be better  if the indexes could be strings and not numbers, but still unnecessary IMO.